### PR TITLE
S1_US1_Navbar

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**
I'm sorry, I suppose that the precedent pull request have been removed when i renamed my branch.

### CODE REVIEW

**Review by Vannou28 — DISMISSED** [review]

File: src/View/component/_navbar.html.twig [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          CONTACT
+      </a>
+    </div>
+  </div>
+  <a class="burger" href="#items">
+    <i class="far fa-eye"></i>
+  </a>
+</nav>
+
+{% block javascript %}<script src="https://kit.fontawesome.com/5b0b013c1e.js" crossorigin="anonymous"></script>{% endblock %}
```
Comment (LucyMerlier) [inline src/View/component/_navbar.html.twig:37]: 2 spaces for indentation? and why not use bootstrap all the way?

File: public/assets/css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    transition: ease 0.7s;
+}
+
+.close {
+    display: none;
+    position:absolute;
+    top: 30%;
+    right: 15%;
+}
+
```
Comment (LucyMerlier) [inline public/assets/css/navbar.css:112]: still a lot of magic values

File: public/assets/css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    .title {
+        letter-spacing: 2px;
+        font-size: 2.6em;
+        padding-top: 4%;
+    }
+
+    .title-box {
+        height:200px;
+    }
+}
```
Comment (LucyMerlier) [inline public/assets/css/navbar.css:150]: are media queries really useful here? if you used bootstrap all the way you may not need media queries

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: public/assets/css/navbar.css (hunk 3 above)
Comment (RaphaelBS-WCS) [reply public/assets/css/navbar.css]: Well, it's ok, the bootstrap navabar is done and all that is no longer there now.

File: public/assets/css/navbar.css (hunk 2 above)
Comment (RaphaelBS-WCS) [reply public/assets/css/navbar.css]: unless I'm mistaken , i've no longer have magic values with my brand new css navbar

File: src/View/component/_navbar.html.twig (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_navbar.html.twig]: Well, the matter come from the visual studio file identifier. I've correct from.

**Review by Vannou28 — APPROVED** [review]
i approuve ;-))

**Review by Vannou28 — APPROVED** [review]
thas is valide!!!

File: public/assets/css/style.css [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+    --light-font-weight:500;
+    --letter-spacing: 10px;
+    --light-letter-spacing:3px;
+
+    --navbar_height: 150px;
+
+    --section_width: 100vw;
+    --width-Big: 90%;
+    --width-Full: 100%;
+    --width-Middle: 40%;
```
Comment (LucyMerlier) [inline public/assets/css/style.css:24]: incoherent naming : use kebab-case

File: src/View/component/_navbar.html.twig [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                <a class="nav-link" href="#">QUI SUIS-JE?</a>
+                <a class="nav-link" href="#">PRESTATIONS</a>
+                <a class="nav-link" href="#">TEMOIGNAGES</a>
+                <a class="nav-link" href="#">CONTACT</a>
+            </div>
+        </div>
+    </div>
+</nav>
+{% endblock %}
+{% block javascript %}<script src="https://kit.fontawesome.com/5b0b013c1e.js" crossorigin="anonymous"></script>{% endblock %}
```
Comment (LucyMerlier) [inline src/View/component/_navbar.html.twig:29]: should add indentation when opening blocks

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: public/assets/css/style.css (hunk 4 above)
Comment (RaphaelBS-WCS) [reply public/assets/css/style.css]: ok we correct it

File: src/View/component/_navbar.html.twig (hunk 5 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_navbar.html.twig]: ok i add it

**Review by Vannou28 — APPROVED** [review]

**Review by Vannou28 — APPROVED** [review]
it's good for me !!!!

**Review by Vannou28 — DISMISSED** [review]
it's good for me !!!

File: src/View/component/_navbar.html.twig [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{% block stylesheet %}
+<link rel="stylesheet" href="/assets/css/navbar.css">
+{% endblock %}
+
+
```
Comment (LucyMerlier) [inline src/View/component/_navbar.html.twig:5]: empty lines?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/component/_navbar.html.twig (hunk 6 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_navbar.html.twig]: erased

**Review by Erdnaxela45 — DISMISSED** [review]
Vraiment incroyable

**Review by Mcbess-san — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**